samples/apm-agent/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it possible to make this feature a module together with metrics to let users dynamically print logs when needed? e.g. use dynamic configuration.

Reply:
yes, will be done in other PR
old metrics only handled two stage of invocation, we will enhance it.